app/src/releasePreprod/kotlin/co/netguru/baby/monitor/client/application/DebugMetricsHelper.kt

```diff
@@ -23,7 +22,6 @@ import javax.inject.Inject
 class DebugMetricsHelper @Inject constructor() {
 
     internal fun init(context: Context) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/1cb717f0cb8fe0d6e255d90e87656ab0740ab3b4/app/src/releasePreprod/kotlin/co/netguru/baby/monitor/client/application/DebugMetricsHelper.kt#L24"></span>Function parameter context is unused.</td>
    </tr>
  </tbody>
</table>